.eslintrc.js

```diff
@@ -1,22 +1,26 @@
 module.exports = {
   root: true,
-  parser: 'babel-eslint',
+  parser: '@typescript-eslint/parser',
   parserOptions: {
     ecmaVersion: 2018,
     sourceType: 'module'
   },
```

Comment:
This ensures editor plugins can run from any CWD and find the right tsconfig.json:

```suggestion
  parserOptions: {
    ecmaVersion: 2018,
    sourceType: 'module',
    project: './tsconfig.json',
    // allows eslint from any dir
    tsconfigRootDir: __dirname,
    sourceType: 'module',
  },
```

Reply:
When I used this we end up with:

```
/Users/ckrycho/dev/ember-modifier/ember-modifier/index.js
  0:0  error  Parsing error: "parserOptions.project" has been set for @typescript-eslint/parser.
The file does not match your project config: index.js.
The file must be included in at least one of the projects provided
```